corehq/apps/cloudcare/templates/formplayer/case_list.html

```diff
@@ -156,16 +156,6 @@
   }
 </script>
 
-<script type="text/template" id="cell-wrapper-style-template">
-  .list-cell-wrapper-style {
-  border: 1px solid white;
-  border-collapse: collapse;
-  vertical-align: top;
-  background-color: #f2f2f1;
-  container-type: inline-size;
-  }
-</script>
```

Comment:
@orangejenny Do you see any issues with moving this to the less file as well? Any thing here will be added to the CSS later meaning it cannot be overridden by anything in the less file.

Reply:
I don't have a strong opinion. Evan's reason for keeping it here was about code organization: https://github.com/dimagi/commcare-hq/pull/32933#discussion_r1190480763

Reply:
@nospame Just taging you that is will move to get the right order in the final CSS for the browser